### ISSUE TITLE
Update text content and format style

### ISSUE THREAD

**Issue by space-partz** [space-partz/Muhammadhakeem_dotcom#56]
### Description

- [ ] We want to clean up the text, include images and icons in place of some titles and headers. The project is too wordy. it needs to be streamlined and simplified. Use the current resume as a source for improved word use.
- [ ] We want to move the search experiences function to the home page. allowing a user to search for the experiences relevant to the visit to the site. We want the title "Experiences" 
- [ ] We are going to create a Figma prototype of the website to fine tune style and design.

### Problem

The site is too wordy. It fatigues me when I look at it. Also, there are some really simple, attractive iterations that can be done right from where we are. 

### Solution Proposal

- [ ] We are going to create a Figma prototype and go from there. By the end of day.
- [ ] We need to write a design document that describes the website.
- [ ] We need to collaborate to carry out the prototype.

### Alternatives Considered

I'm not going to look at the prototype like a graphic project. It is still a majority text project and does not need the time-consuming graphic design relevant to other projects. My plan is to create text blocks with icons to break up the monotony. Rearrange the text bubbles until I am satisfied. 3hrs

### Additional Context

Check out these sources for inspiration: https://weirdweboctober.website/, https://www.joshuabrigati.com/

**Comment by azarah** [comment]
**Description Cont'd**
_"We want to move the search experiences function to the home page. allowing a user to search for the experiences relevant to the visit to the site. We want the title "Experiences"_

In place of the title **Experiences** will be a Search Button with an Input function that will allow the user to search the site for _relevant information_ that best _represents the experiences_ the user is after.
_*Note: The size of the New Experiences section on the website must be consistent for whatever size will be the Skills section and Education Section._

- When the user first visits the site, they will be able to view a Search button with Input Option 
- After the user Enters the Search Term, they will able to select from a list of beautifully formatted information from across the site. Similar to how Google formats theirs but beautifully themed. We may even implement its protocol in our site.

Overall I believe this level of functionality will encourage the page to look minimalist on the front-end however reveal a level of detail taken into consideration on the back-end to reflect the functionality of attempting to simplify the needs of potential clients to view immediate solutions to their problems and be guided along the way.
- [ ] Create an Experiences Container to house the experiences features

- [ ] Create a Search Button: _Search Experiences..._Another Button that says "**Search**/Enter(Search turns into Enter after term in input)/**Clear**(after a term is input)". However the user will also be able to hit **Enter** on Keyboard 

- [ ] Create Script that will search entire site to represent what Client is looking for based on SEO, Meta Data, Alt tags.

FURTHER, While writing this, I have a theory that may work for this situation which is, create a personal site dictionary that the script is able to use as a cross reference to compute terms similar to what the client is searching for. This will allow the author to update or amend to terms present in the site without changing the site in order to keep up with trends, buzz words and lingo and different languages.
